src/uct/api/uct.h

```diff
@@ -258,9 +259,25 @@ typedef struct uct_component_attr {
      */
     uct_md_resource_desc_t *md_resources;
 
+    /**
+     * Flags as defined by UCT_COMPONENT_FLAG_xx.
+     */
+    uint64_t               flags;
 } uct_component_attr_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Capability flags of @ref uct_component_h.
```

Comment:
Please add full description. Just "brief" looks bad in generated documents.

Reply:
added

Reply:
@shamisp pls take a look

Reply:
Looks good. thanks